files/grest/rpc/blockchain/account_balance.sql

```diff
@@ -1,3 +1,5 @@
+DROP FUNCTION IF EXISTS grest.account_balance (text);
+
 CREATE OR REPLACE FUNCTION grest.account_balance (_address text)
```

Comment:
Insignificant but if droped first then OR REPLACE can be removed as well.

Reply:
good point, added commit to keep all queries uniform